ch_pipeline/analysis/flagging.py

```diff
@@ -210,6 +210,44 @@ def _static_rfi_mask_hook(self, freq, timestamp=None):
         return ~rfi.frequency_mask(freq, timestamp=timestamp)
 
 
+class MaskStaticRFI(task.SingleTask):
```

Comment:
Should this be called `StaticRFIMask` or something similar, since it generates a mask but doesn't actually apply the mask?

Reply:
Or maybe `GetStaticRFIMask`...

Reply:
I guess the current standard is `RFI___Mask` so I've renamed it to follow that